sys/auto_init/auto_init.c

```diff
@@ -28,6 +28,11 @@
 
 void auto_init(void)
 {
+    if (IS_USED(MODULE_AUTO_INIT_LEDS)) {
+        LOG_DEBUG("Auto init LEDs.\n");
+        extern void leds_int(void);
+        leds_int();
```

Comment:
let's better move this to `drivers/periph_common/init.c`, otherwise it happens way too late.

Ideally we'd create a new function `early_init()` that calls 

    leds_int();
    stdio_init();
    periph_init();

since LEDs will always be the simplest debugging tool

Reply:
Makes sense!